source/administration/devops-command-center.rst

```diff
@@ -62,50 +62,124 @@ The set of steps contained within a playbook is called a checklist.
+- **Private:** Only incident members who were members of the channel when the incident started have access to the ongoing incident and incident details after the incident has ended. Private incidents can't be made public once they've been started.
+
+Private incidents don't appear in the backstage when they're active, and the contents of the channel won't show up in a search query. If an incident was public and changed to private, it can't be accessed by non-participants unless they're invited by a member of the incident channel.
+
+**Creating a public playbook and incident**
+
+1. Navigate to **Main Menu > Incidents & Playbooks > Playbooks**.
+2. Select **+ New Playbook**.
+3. Enter a name for the playbook.
+4. Move the toggle to the right to **Create Public Incident**.
+5. Add a few checklist items.
+6. Select **Save**.
+7. Switch to the main channel view.
+8. Create an incident by selecting the playbook you just created.
 
-You can add steps and edit checklist items during an active incident. You can also edit checklist items in playbooks that are not being used in active incidents.
+**Creating a private playbook and incident**
 
-To edit a checklist, choose **Edit**.
+1. Navigate to **Main Menu > Incidents & Playbooks > Playbooks**.
+2. Select **+ New Playbook**.
+3. Enter a name for the playbook.
+4. Keep the **Create Public Incident** option disabled.
+5. Add a few checklist items.
+6. Select **Save**.
+7. Switch to the main channel view.
+8. Create an incident by selecting the playbook you just created.
 
-- Select the text field to edit the item details.
-- Hover over an input box and select the **X** to remove the checklist item.
-- Drag the hamburger menu to rearrange the checklist items.
+Editing checklists
+~~~~~~~~~~~~~~~~~~~
+
+You can edit checklist items and their position in the checklist when they are not part of an active incident.
 
-Choose **Done** to save the changes.
+1. Select **!** in the channel header of an active incident.
+2. Choose **Edit**.
+  - To edit: Select the text field to edit details.
+  - To delete: Hover over an input box and select the **X** to remove the checklist item.
+  - To move: Drag the hamburger menu to rearrange the checklist items.
+3. Choose **Done** to save the changes.
 
 Ending incidents
 ~~~~~~~~~~~~~~~~
 
 There are two ways to end an incident:
 
-- Using slash command */incident end* from within the incident channel.
+- Using the slash command */incident end* from within the incident channel.
 - Using the **End Incident** button in the RHS panel while in the incident channel.
 
-The incident will become inactive, be removed from the list of active incidents, and the associated channel will be archived.
+The incident will become inactive, is removed from the list of active incidents, and the associated channel is archived.
```

Comment:
Let's remove this paragraph, since there is no longer a list of active incidents, and we don't automatically archive the channel.
```suggestion
```